tests/drum/custom_java_predictor/pom.xml

```diff
@@ -26,9 +26,9 @@
             <version>2.7</version>
         </dependency>
         <dependency>
-            <groupId>com.datarobot.drum</groupId>
-            <artifactId>predictors</artifactId>
-            <version>1.0</version>
+            <groupId>com.datarobot</groupId>
+            <artifactId>drum-base-predictor</artifactId>
+            <version>1.1.0</version>
```

Comment:
hm... `rum-base-predictor` version is `1.0.0` in `custom_model_runner/datarobot_drum/drum/language_predictors/java_predictor/base_predictor/pom.xml` and as a dependency in all other pom files. 
How did this build?

Reply:
I noticed and update this :)